src/servers/event-handlers/kratos.ts

```diff
@@ -31,11 +32,11 @@ kratosCallback.post(
     fn: async (req: express.Request, res: express.Response) => {
       const secret = req.headers.authorization
       const body = req.body
-      addAttributesToCurrentSpan({
```

Comment:
for some reasons, this span does show up

Reply:
maybe because `addAttributesToCurrentSpan` inside `wrapAsyncToRunInSpan` ? haven't tried to debug it too much